build.gradle

```diff
@@ -96,6 +96,7 @@ allprojects {
   group = "org.apache.iceberg"
   version = projectVersion
   repositories {
+    maven { url  "https://repository.apache.org/content/repositories/orgapachespark-1418/" }
```

Comment:
Will remove it if Spark 3.3.1 is officially released.

Reply:
@Fokko Do you know how to run a single test? I'd like to run these tests locally:
```
org.apache.iceberg.spark.extensions.TestRewriteDataFilesProcedure > testRewriteDataFilesOnNonPartitionTable[catalogName = testhive, implementation = org.apache.iceberg.spark.SparkCatalog, config = {type=hive, default-namespace=default}] FAILED
    java.lang.AssertionError: Action should rewrite 10 data files and add 1 data files: row 1 col 1 contents should match expected:<10> but was:<7>
        at org.junit.Assert.fail(Assert.java:89)
        at org.junit.Assert.failNotEquals(Assert.java:835)
        at org.junit.Assert.assertEquals(Assert.java:120)
        at org.apache.iceberg.spark.SparkTestBase.assertEquals(SparkTestBase.java:181)
        at org.apache.iceberg.spark.SparkTestBase.assertEquals(SparkTestBase.java:163)
        at org.apache.iceberg.spark.extensions.TestRewriteDataFilesProcedure.testRewriteDataFilesOnNonPartitionTable(TestRewriteDataFilesProcedure.java:104)

org.apache.iceberg.spark.extensions.TestRewriteDataFilesProcedure > testRewriteDataFilesWithZOrder[catalogName = testhive, implementation = org.apache.iceberg.spark.SparkCatalog, config = {type=hive, default-namespace=default}] FAILED
    java.lang.AssertionError: Action should rewrite 10 data files and add 1 data files: row 1 col 1 contents should match expected:<10> but was:<7>
        at org.junit.Assert.fail(Assert.java:89)
        at org.junit.Assert.failNotEquals(Assert.java:835)
        at org.junit.Assert.assertEquals(Assert.java:120)
        at org.apache.iceberg.spark.SparkTestBase.assertEquals(SparkTestBase.java:181)
        at org.apache.iceberg.spark.SparkTestBase.assertEquals(SparkTestBase.java:163)
        at org.apache.iceberg.spark.extensions.TestRewriteDataFilesProcedure.testRewriteDataFilesWithZOrder(TestRewriteDataFilesProcedure.java:171)
```

Reply:
@wangyum The easiest way is to use IntelliJ and just run a single test from the UI:
![image](https://user-images.githubusercontent.com/1134248/193600165-46b40dfe-b352-4c67-87c8-a86d5afbfc52.png)